fxgl/src/test/kotlin/com/almasb/fxgl/dsl/components/ProjectileComponentTest.kt

```diff
@@ -34,6 +35,40 @@ class ProjectileComponentTest {
         assertThat(comp.direction, `is`(Point2D(1.0, 0.0)))
         assertThat(comp.speed, `is`(100.0))
         assertThat(comp.velocity, `is`(Point2D(100.0, 0.0)))
+        assertThat(comp.acceleration, `is`(Point2D(0.0, 0.0)))
+    }
+
+    @Test
+    fun `Creation with acceleration`() {
+        comp = ProjectileComponent(Point2D(0.0, 0.0), 0.0, Point2D(1.0, 1.0))
+        e.addComponent(comp)
+
+        assertThat(comp.acceleration, `is`(Point2D(1.0, 1.0)))
+        assertThat(comp.direction, `is`(Point2D(0.0, 0.0)))
+        assertThat(comp.speed, `is`(0.0))
+    }
+
+    @Test
+    fun `Acceleration change direction and speed`(){
+        val tpf = .01
+        val acceleration = Point2D(100.0, 0.0)
+
+        comp = ProjectileComponent(Point2D(0.0, 0.0), 0.0, acceleration)
+        e.addComponent(comp)
+
+        assertThat(comp.acceleration, `is`(acceleration))
+
+        comp.onUpdate(tpf)
+
+        assertThat(comp.direction, `is`(acceleration.normalize()))
```

Comment:
1. Please update assertions of `direction` and `speed`, using hardcoded expected values to be consistent with other tests, e.g.:

```
assertThat(comp.direction, `is`(Point2D(1.0, 0.0)))
```

Same for lines 69-70.

2. We also need to test the entity position, after each `comp.onUpdate()`, e.g.:

```
assertThat(e.position, `is`(Point2D(1.0, 0.0)))
```

Reply:
Done